app/styles/ember-appuniversum/_c-tabs.scss

```diff
@@ -9,12 +9,12 @@
   padding: 0 $au-unit;
 
   &:not(.au-c-tabs--reversed) {
-    border-bottom: .1rem solid var(--au-gray-300);
```

Comment:
Maybe this is a good opportunity to add a new css variable for this so new components can use that as well? It would make changing the border color in all components easier as well?

Reply:
Yes, we could do that. Would prefer to make a separate PR for this as this needs an appuniversum release. I would probably then look into adding some more global variables.